crates/rbuilder/src/live_builder/payload_events/mod.rs

```diff
@@ -30,11 +30,14 @@ const NEW_PAYLOAD_RECV_TIMEOUT: Duration = SLOT_DURATION.saturating_mul(2);
 /// One slot (12secs) is enough so we don't saturate any resource and we don't miss to many slots.
 const CONSENSUS_CLIENT_RECONNECT_WAIT: Duration = SLOT_DURATION;
 
+/// Data about a slot received from relays.
+/// Contains the important information needed to build and submit the block.
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct MevBoostSlotData {
+    /// The .data.payload_attributes.suggested_fee_recipient is replaced
     pub payload_attributes_event: PayloadAttributesEvent,
     pub suggested_gas_limit: u64,
-    /// List of relays that have this slot registered
+    /// List of relays that have this slot registered.
```

Comment:
this wording isn't great.. maybe "List of relays registered for this slot"? not sure about the semantics.